admin/README.md

```diff
@@ -0,0 +1,14 @@
+# Usage
+
+Each script can be executed using Nix with:
+
+```
+nix run .#<script-name>
+```
+
+If you activate the development environment with `nix develop`, all scripts
```

Comment:
What about printing the list of the scripts when user enters `nix develop`, so the know what they are ?